src/routes/transactions/mappers/common/transaction-info.mapper.ts

```diff
@@ -132,13 +151,17 @@ export class MultisigTransactionInfoMapper {
     value: number,
     dataSize: number,
   ): boolean {
+    const isSettingsChange: boolean = transaction.dataDecoded
```

Comment:
nitpick: what about renaming this `isSettingsChangeMethod` to avoid colliding the containing function name?

Reply:
Changed in [7ed250e](https://github.com/5afe/safe-client-gateway-nest/pull/245/commits/7ed250ebe5aefdbd2a26d74cf769acd303da7e77)